src/components/routes/AppRoutes.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { useRoutes } from "react-router-dom";
+
+import Landing from "./Landing";
```

Comment:
can you create an index file for this kind of components? 

and import those like 

`import { Landing } from 'components'; `